README.md

```diff
@@ -1,32 +1,44 @@
 Node Apps Toolkit
 ===
 
-A collection of helpers and utilities for creating NodeJS Contentful Apps
+The Node Apps Toolkit is a growing collection of helpers, and utilities, for creating NodeJS Contentful Apps.
 
 ## Getting started
  
-You can install this library via
+You can install this library with npm or yarn
  
 ```
 npm install --save @contentful/node-apps-toolkit
+or
+yarn add @contentful/node-apps-toolkit
 ```
 
-and include it in your code like
+## Features
 
+### getManagementToken
+NodeJS Contentful Apps need a management token to interact with Contentful's APIs.
+Creating a management token requires a key pair to be regsitered for the app, follow [this link](http://contentful./developers/docs/references/content-management-api/#/reference/app-keys/app-keys) for more information on key pairs.
+Once a key pair is registered the getManagementToken function can be used to generate a valid token.
+
+This example demonstrates how to import and use `getManagementToken`
 ```js
-const {getManagementToken} = require('@contentful/node-apps-toolkit');
-const {appInstallationId, spaceId, privateKey} = require('./some-constants');
+import { getManagementToken } from '@contentful/node-apps-toolkit';
+import { appInstallationId, spaceId, privateKey } from './some-constants';
 
 getManagementToken(privateKey, {appInstallationId, spaceId})
     .then((token) => {
       console.log('Here is your app token')
       console.log(token)
     })
 ```
+Once a token has been obtained, it can be used to make requests to Contentful's API.
+
+## More coming soon
+We're excited to expand this toolkit with new features. If you have any suggestions or requests for features you'd like to see please create an issue in this repo!
 
-## API Docs
+## API Documentation
 
-API documentation is available [here](https://contentful.github.io/node-apps-toolkit/)
+In depth API documentation is available [here](https://contentful.github.io/node-apps-toolkit/)
 
 ## Testing
```

Comment:
Since we're at it, maybe we can go a little bit in depth about testing: what commands we should run for the tests, what is the setup required and how do I fetch the information to populate the `.env` file. I did a terrible job in documenting this part in the first release, I am very sorry 😅